homeassistant/components/evohome/__init__.py

```diff
@@ -674,8 +692,8 @@ def _dt_evo_to_aware(dt_naive: dt, utc_offset: timedelta) -> dt:
                 )
 
                 self._setpoints[f"{key}_sp_from"] = dt_aware.isoformat()
-                try:
-                    self._setpoints[f"{key}_sp_temp"] = switchpoint["heatSetpoint"]
+                try:  # heatSetpoint is the only pascalCase string
```

Comment:
The comment is confusing as heatSetpoint is camelCase.

Reply:
yes - this is a typo - even the intended statement: "heatSetpoint is the only camelCase string" is confusing. I'll just remove it.

The mixture of the two schemes is due to an anachronism from when the legacy client library was ported over to aiohttp. I'll convert them all over to snake_case shortly (this won't be a breaking change).

I have just re-checked - the strings are the correct case - it is only the comment that is wrong.